website/views/remote/remote.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+from django.http import HttpResponse
+from django.core.paginator import Paginator, InvalidPage, EmptyPage
+from django.template.loader import render_to_string
+from rest_framework.decorators import api_view
+
+from website.decorators import room_required
+from website.json_renderer import JSONResponse
+
+from music.serializers import MusicSerializer
+
+
+@api_view(['POST'])
+@room_required
+def volume_change(request, room):
+    if room.current_music:
+        if request.data.get('volume_change') == 'up':
+            room.increase_volume()
+        else:
+            room.decrease_volume()
+        return HttpResponse(204)
+    return HttpResponse(404)
+
+
+@api_view(['POST'])
+@room_required
+def trigger_shuffle(request, room):
+    if not request.data.get('shuffle') or room.music_set.count() == 0:
+        return HttpResponse(409)
+
+    room.toggle_shuffle((request.data.get('shuffle') == 'true'))
+    remote_template_rendered = render_remote(room=room)
+    return JSONResponse(remote_template_rendered)
+
+
+@api_view(['POST'])
+@room_required
+def next_music(request, room):
+    if not room.current_music or request.data.get('music_id') == room.current_music.music_id:
```

Comment:
if there is no room.current_music then room.current_music has no attibrute music_id...

it happend when i reloaded the server (force current_music to none) but not refreshed the website and clicked on next